lib/models/articles.js

```diff
@@ -0,0 +1,24 @@
+const { getCollection } = require("../database");
+
+async function readArticles() {
+  const articleCollection = await getCollection("articles");
+  const articles = await articleCollection.find({}).toArray();
+
+  if (!articles) {
+    return null;
+  }
```

Comment:
When do you expect to have this case?
When is `articles` falsy?

Reply:
Thanks for the comment!